src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -743,6 +743,61 @@ int shim::p2pEnable(bool enable, bool force)
     return p2p_enable;
 }
 
+int shim::cmaEnable(bool enable, uint64_t size, uint64_t num, bool force)
+{
+    const std::string input = "1\n";
```

Comment:
Let's move all this declarations and definitions to the scope where they are used.  If you do so, you will find that at least one of these variables is not used at all.